src/Features/Core/Portable/CodeRefactorings/FixAllOccurences/IFixAllCodeRefactoringGetFixesService.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    internal interface IFixAllCodeRefactoringGetFixesService : IWorkspaceService
+    {
+        /// <summary>
+        /// Computes the fix all occurrences code fix, brings up the preview changes dialog for the fix and
+        /// returns the code action operations corresponding to the fix.
+        /// </summary>
+        Task<ImmutableArray<CodeActionOperation>> GetFixAllOperationsAsync(FixAllContext fixAllContext, bool showPreviewChangesDialog);
+
+        /// <summary>
+        /// Computes the fix all occurrences code fix and returns the changed solution.
```

Comment:
what does returning null mean here?

Reply:
Null means no changes. This is based on the already annotated method: https://github.com/dotnet/roslyn/blob/315c2e149ba7889b0937d872274c33fcbfe9af5f/src/Workspaces/Core/Portable/CodeActions/CodeAction.cs#L189